dataverse-ext-common/src/main/java/edu/harvard/iq/dataverse/importer/metadata/ImporterConstants.java

```diff
@@ -3,5 +3,6 @@
 public class ImporterConstants {
     public static final String IMPORTER_NAME = "importer.name";
     public static final String IMPORTER_DESCRIPTION = "importer.description";
-    public static final String UNKNOWN_BUNDLE_VALUE = "???";
+
+  public static final String FILE_NAME_SEPARATOR = "---";
```

Comment:
For better scoping & encapsulation I'd rather define it as a private constant in a class that actually uses it, rather then a public field somewhere else. Especially that it's used only in a single place.

Reply:
Yes, but in this case I imagine that it will be used not only by the class where it's currently used, but also by the importer to extract the proper file name.

Reply:
YAGNI? You can always up-scope if/when you'll actually going to need that?

Reply:
I'm not sure if we can talk about YAGNI here. That was not done as something that "perhaps could turn out to be useful" but rather as something that "definitely be needed" for filenames handling.